### DISCUSSION TITLE
How to restrict the web interface to HTTPS?

### DISCUSSION THREAD

**Discussion by ifrx** [datenstrom/community#336] [Ask a question] [answered]
For my Yellow 0.7.4 instance is accessible via different TLDs where only one TLD is https secure, I would love to restrict the Webinterface or Editor to https access.

As @markseu mentioned [a while ago](https://github.com/datenstrom/yellow/issues/6#issuecomment-59489874), the "webinterfaceServerScheme" variable could do but that option does not seem to be available anymore.

How can I restrict the Webinterface or Editor accessibility to https today?

**Comment by markseuffert** [comment, marked as answer]
I am sorry, this is not possible. The current software doesn't support this feature anymore. Now the editor behaves like the rest of the website. It's best if your website automatically redirects from HTTP to HTTPS and the Internet connection is always encrypted.

See also: How to redirect all HTTP requests to HTTPS - #272

**Comment by ifrx** [comment]
Good to know. Will move to a hosting service with multi-domain SSL. Thank you for clarifying that.
